### ISSUE TITLE
Drag file on Breadcrumbs ?

### ISSUE THREAD

**Issue by Carreau** [jupyter/jupyterlab#598]
Trying to force myself to use the filebrowser, it's really intuitive. 
I wanted to move the file to a sibling directory, so I was wondering how to do that. 

I can see a few ways:
  - Allow 2 side by side filebrowser
  - allow to drop file on breadcrumbs
  - if dragging on a breadcrumb for a  long time, navigate automatically to this location, 
    - same applies if hovering a folder for a long time 

Basically a time-based context-action. 

My guess is that it could generally be useful to have long-hover / long press of element triggering alternate actions for let say touch-based interface. 

Thoughts ?

**Comment by blink1073** [comment]
We can already drop file(s) on breadcrumbs.  For sibling folder moving, you can `cut` and then `paste` the items.

I am `-1` on any magical navigation.  I'd prefer to show a tooltip on hover, which is what the user will most likely be expecting.

**Comment by jasongrout** [comment]
FYI, OS X implements this long-hover-to-open-folder and it works nicely when dragging files around. See https://www.youtube.com/watch?v=gdUPrMjlLy8, for example.

**Comment by blink1073** [comment]
Cool, that does seem pretty intuitive.

**Comment by sccolbert** [comment]
Yeah, long-hover-to-open is a nice feature, but can be tricky to implement. I'm not opposed to this, but it may be involved to get right.

**Comment by blink1073** [comment]
Yep, notice I didn't offer to do it right now ;).

**Comment by jasongrout** [comment]
For reference, "long-hover-to-open" is called spring-loaded folders in OS X....

**Comment by ellisonbg** [comment]
We are fine with this feature being implemented as long as it is done well.

**Comment by tgeorgeux** [comment]
This feature seems helpful, for implementation, I believe we should make sure if we are using spring-loading for breadcrumbs we should also use it for all folders in the File Browser.
